version.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "2.5",
+  "publicReleaseRefSpec": [
+    "^refs/heads/master$"
+  ],
+  "buildNumberOffset": 3,
```

Comment:
📝 This accounts for the fact that you were already making releases with version numbers **2.5.x**. The property can be permanently removed starting with your first **2.6+** build.